open_src/influx/meta/data.go

```diff
@@ -1415,6 +1415,25 @@ func (data *Data) UpdateRetentionPolicy(database, name string, rpu *RetentionPol
 	return nil
 }
 
+// ShowContinuousQueries shows all continuous queries group by db.
+func (data *Data) ShowContinuousQueries() (models.Rows, error) {
```

Comment:
Add test case

Reply:
I have added the corresponding test case.